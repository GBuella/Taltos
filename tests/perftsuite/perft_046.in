setboard B6b/8/8/8/2K5/4k3/8/b6B w - - 0 1
echo B6b/8/8/8/2K5/4k3/8/b6B w - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
