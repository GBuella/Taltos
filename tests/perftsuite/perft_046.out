B6b/8/8/8/2K5/4k3/8/b6B w - - 0 1
17
278
4607
76778
1320507
22823890
